src/pyff/samlmd.py

```diff
@@ -1047,14 +1047,25 @@ def discojson_sp_attr(e):
     if b64_trustinfos is None:
         return None
 
+    entityID = e.get('entityID', None)
+    if entityID is None:
+        return None
```

Comment:
Why would an entry have no entityID?
Isn't this check part of a validation step already, while ingesting input?

Reply:
Yeah you are right, I'll remove that